src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -41,6 +43,11 @@ public void driveSystem() {
   public void driveSystem(double speed, double zRotation) {
     maindrive.arcadeDrive(speed, zRotation,  false);
   }
+  
+  protected void execute() {
+    SmartDashboard.putBoolean("Bridge Limit", bridge);
+  }
+    boolean bridge = true;
```

Comment:
Changes were added to this file that doesn't need to be here (for this issue).  Remove it.